src/pybind/mgr/cephadm/module.py

```diff
@@ -2097,39 +2137,30 @@ def _create_daemon(self, daemon_type, daemon_id, host,
 
         start_time = datetime.datetime.utcnow()
         deps = []  # type: List[str]
-        cephadm_config = {}  # type: Dict[str, Any]
+        cephadm_config = {} # type: Dict
```

Comment:
why?

Reply:
mistake during rebase/fixup ... it's reverted